packages/@react-aria/dnd/src/useDraggableItem.ts

```diff
@@ -36,6 +37,7 @@ export function useDraggableItem(props: DraggableItemProps, state: DraggableColl
     preview: state.preview,
     getAllowedDropOperations: state.getAllowedDropOperations,
     onDragStart(e) {
+      setDraggingCollectionRef(props.parentRef);
```

Comment:
Create useDraggableCollection so we don't have to pass the parentRef to useDraggableItem